src/index.html

```diff
@@ -149,6 +156,44 @@ <h1 class="profile__title">Jacques Cousteau</h1>
         ></p>
       </div>
     </div>
+    <!--Delete Card Modal-->
+    <div class="modal" id="delete-modal">
+      <div class="modal__container">
+        <button class="modal__close"></button>
+        <form class="modal__form">
+          <p class="modal__heading">Are you sure?</p>
+          <button class="modal__button">Yes</button>
+        </form>
+      </div>
+    </div>
+    <!--Change Profile Picture Modal-->
+    <div class="modal" id="change-modal">
+      <div class="modal__container">
+        <button class="modal__close"></button>
+        <p class="modal__heading">Change profile picture</p>
+        <form
+          class="modal__form"
+          id="changePictureForm"
+          name="cardForm"
```

Comment:
`name="cardForm"` is already in line 117



This is for the `avatar` form, so it should be `avatarForm`